### PR TITLE
:sparkles: Trigger a manual test on crowd-testing provider TestIO

### PR DESCRIPTION
**Pull request by jreimone**
### Type of Change

- [ ] Bugfix
- [x] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

We want to enable everyone to trigger a manual test on the crowd-testing provider TestIO from a PR.

### Checklist

- [x] Add relevant labels (for example type of change or patch/minor/major)
- [x] Make sure not to introduce some mistakes
- [ ] Update documentation
- [x] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Comment by jreimone** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jreimone** [comment]
recheck

File: .github/workflows/template_testio_trigger_test.yml
Comment (flaxel) [inline .github/workflows/template_testio_trigger_test.yml:28]: Will there be a release working flow in the future?! 🤔

File: .github/workflows/template_testio_trigger_test.yml
Comment (flaxel) [inline .github/workflows/template_testio_trigger_test.yml:1]: Please add a new entry to the README. 🙌

File: .github/workflows/template_testio_trigger_test.yml
Comment (jreimone) [reply .github/workflows/template_testio_trigger_test.yml:28]: Release drafter already set up, but no release yet since documentation is still missing in the action repo. But yes, we will release and reflect that here of course 👍

File: .github/workflows/template_testio_trigger_test.yml
Comment (jreimone) [reply .github/workflows/template_testio_trigger_test.yml:1]: Already planned

File: .github/workflows/template_testio_trigger_test.yml
Comment (0x46616c6b) [inline .github/workflows/template_testio_trigger_test.yml:28]: Is the plan to open source the `Staffbase/testio-trigger-test-github-action` action in future? Otherwise I would argue against having a public reusable workflow with a private action.

Comment (flaxel) [reply .github/workflows/template_testio_trigger_test.yml:28]: It will be public in the future. 🙌

File: README.md
Comment (flaxel) [inline README.md:311]: I would make it consistent and we only add the optional information for the other workflows. 
```suggestion
      # ID of the product on the TestIO platform to which the triggered test should be assigned to
```

File: README.md
Comment (flaxel) [inline README.md:303]: I guess there is also some logic that you don't spam the TestIO people? 🤔

File: .github/workflows/template_testio_trigger_test.yml
Comment (jreimone) [reply .github/workflows/template_testio_trigger_test.yml:28]: Yes, once [this Legal](https://mitarbeiterapp.atlassian.net/servicedesk/customer/portal/8/LP-870) request got approved.

File: README.md
Comment (flaxel) [reply README.md]: Please also for the other comments. 😆

File: README.md
Comment (jreimone) [reply README.md:303]: We pay good money 💸 So I don't get your point 😉

**Review by flaxel — APPROVED** [review]

**Comment by jreimone** [comment]
@flaxel thanks for your review 🙏 What do you suggest, shall I merge this PR now or wait until I have the approval from Legal to open-source the GHA?

**Comment by flaxel** [comment]
@jreimone I guess it's valid to merge the PR now depending on how urgent it is to bring it to production. 🙌

**Review by HarshStaffbase — APPROVED** [review]

**Comment by HarshStaffbase** [comment]
💙

**Pull request merged**
